KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -232,15 +234,14 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
     }
     
     override func updateViewConstraints() {
```

Comment:
This whole to overriding `updateViewConstraints`/`updateConstraints` is not needed and do not know why someone decided to create it like that. Just saying, nothing to fix here for you.